src/rusoto_dynamodbstore.rs

```diff
@@ -14,32 +14,13 @@ use std::{
 };
 use tracing::{field::Empty, info_span, Span};
 
-fn add_err_to_span(e: &(dyn std::error::Error + Sync + Send + 'static)) {
-    let span = Span::current();
-    span.record("otel.status_code", "ERROR");
-    span.record("error.msg", e);
-}
-
 trait ErrExt {
     fn spanify(self) -> Self;
 }
 
 impl<E: std::error::Error + Sync + Send + 'static> ErrExt for E {
     fn spanify(self) -> Self {
```

Comment:
This diff is inaccurate, I just moved `ResultExt` entirely out of the file and its struggling to understand...